### ISSUE TITLE
Certain screen readers are unable to edit numeric input fields 

### ISSUE THREAD

**Issue by toddparker** [filamentgroup/formcore#89]
On Chrome, Certain screen readers are unable to edit numeric input fields 
For example, the credit card on this screen and the zip field on the prior screen. JAWS for Windows reports these fields as an edit spin box. I am able to type the number, but am unable to edit the number once inputted or review the typed numbers. This will be true for all fields that use this same type of input control.
